project-progression-policy.md

```diff
@@ -145,5 +145,29 @@ Projects may be granted Emeritus status via a vote of two-thirds of the represen
 
 ## IV. Annual Review Process
 
-The TAC shall develop an annual review process to determine whether projects are in the stage that accurately reflects their needs and goals. 
+The TAC will conduct a review of each accepted project on an annual basis.
 
+The review includes the following:
+
+1. Review whether any answers to the project's submission template have
```

Comment:
Not all answers are relevant to a running project.  Once the template is complete, we should consider which ones we think should be reviewed.

Reply:
@MikeCamel Sure, but if its not relevant after creation, then the answer won't have "changed" per se so I don't think the text here is incorrect, do you?   Just a request for a future action item I think once we have more clarity.